backend/common/src/main/java/ai/verta/modeldb/common/authservice/RoleService.java

```diff
@@ -53,11 +53,6 @@ List<GetResourcesResponseItem> getResourceItems(
       ModelDBServiceResourceTypes modelDBServiceResourceTypes,
       boolean isServiceUser);
 
-  List<GetResourcesResponseItem> getResourceItemsSpecialPersonalWorkspace(
```

Comment:
Will removing this cause any issues for environments that don't have the new feature flag enabled?

Reply:
Actually, it was not used anywhere so I have removed it. it will not cause any issue.